content/topics/solo-learn/sql/introduction-to-sql/1-getting-started-with-sql/_index.md

```diff
@@ -0,0 +1,24 @@
+---
+_db_id:
+content_type: topic
+ready: true
+tags:
+- sql
+- solo-learn
+title:Solo Learn - Introduction to SQL - 1 Getting Started with SQL

```

Comment:
Please run Hugo locally to avoid this issue. Check if the server is running again, as these errors will prevent the site from rendering locally. https://github.com/Umuzi-org/ACN-syllabus?tab=readme-ov-file#running-locally